src/mqueryfront/src/QueryResultsStatus.js

```diff
@@ -257,6 +257,12 @@ class QueryResultsStatus extends Component {
                         >
                             {this.props.job.status}
                         </span>
+                        {this.props.job.total_files - lenMatches > 0 && (
```

Comment:
I'd prefer to have the number of errored samples on the progress bar (like the samples in progress, but in red).